.github/workflows/windows.yml

```diff
@@ -76,7 +77,7 @@ jobs:
                 -D SENTRY_DSN=${{ secrets.SENTRY_DSN }} \
                 -D SENTRY_ENV="${APP_ENV}" \
                 -D CMAKE_MSVC_RUNTIME_LIBRARY="MultiThreaded\$<\$<CONFIG:Debug>:Debug>" \
-                -D PKG_CONFIG_EXECUTABLE=${CMAKE_BUILD_DIR}/vcpkg_installed/x64-windows/tools/pkgconf/pkgconf.exe \
+                -D PKG_CONFIG_EXECUTABLE=${CMAKE_BUILD_DIR}/vcpkg_installed/x64-windows-static/tools/pkgconf/pkgconf.exe \
```

Comment:
@m-kuhn , wondering if it'd be worth using a variable for the x64-windows-static part of this path (as well as for VCPKG_TARGET_TRIPLET and VCPKG_HOST_TRIPLET). I'm worried we'll end up forgetting about it and if one day we change the triplet cmake file name, we'll end up spending time debugging a mysterious failure :)

Reply:
If "making a variable" would be very straightforward and maintainable with github actions I would make things considerably more DRY in general. I'm sure if we rename triplets we will find that out ;-)

Reply:
@m-kuhn , OK, auto merge enabled.